FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_keep_alive/DemoTasks/KeepAliveMQTTExample.c

```diff
@@ -182,23 +185,46 @@
  */
 static void prvMQTTDemoTask( void * pvParameters );
 
+/**
+ * @brief Connect to MQTT broker with reconnection retries.
+ *
+ * If connection fails, retry is attempted after a timeout.
+ * Timeout value will exponentially increase until maximum
+ * timeout value is reached or the number of attempts are exhausted.
+ *
+ * @param pxNetworkContext The output parameter to return the created network context.

```

Comment:
@param[out] would have been a better way to mention this in the documentation. I see that the demo follows this style through out, it may be a bigger change to make.

Reply:
yea the trouble is that only the mutual auth demos use @param[in/out] while the rest use @param

Reply:
How about creating  a task to take care of it later? I think we need to be consistent in our doc style.

Reply:
I will make a task for this